core/chains/evm/config/config.go

```diff
@@ -94,6 +94,7 @@ type GasEstimator interface {
 	PriceMax() *assets.Wei
 	PriceMin() *assets.Wei
 	Mode() string
+	MaxPriceKey(gethcommon.Address) *assets.Wei
```

Comment:
Can we align the naming one way or another? e.g. `PriceMax` not `MaxPrice`?

I think individual methods could become a mess though, if we add more over time. It may be cleaner to have a `KeySpecific` method that just returns a normal `GasEstimator` interface, backed by the key specific values. :thinking:

Reply:
Changed to `PriceMaxKey` in [`934bbe1` (#9694)](https://github.com/smartcontractkit/chainlink/pull/9694/commits/934bbe1ec62c8430ae7fbd72644da7c7d7230090).
@jmank88 I agree things could get messy in the future if we add more key specific functionalities, but we should make the decision when the need arises, as we would have more context to make a better decision